db/migrate/20240712154910_create_benefits_intake_form_submissions.rb

```diff
@@ -0,0 +1,19 @@
+class CreateBenefitsIntakeFormSubmissions < ActiveRecord::Migration[7.1]
+  def change
+    create_table :benefits_intake_form_submissions do |t|
+      t.string :form_type, null: false
+      t.uuid :benefits_intake_uuid
+      t.uuid :submitted_claim_uuid
```

Comment:
Do we still need this?

Reply:
I do not believe so.  could we also drop `in_progress_form`?

`saved_claim_id` is automagically added because this references `:saved_claim`?

Reply:
> could we also drop `in_progress_form`?

I think so. Our current usage of `FormSubmission` leads me to believe we aren't using it anyway.

> `saved_claim_id` is automagically added because this references `:saved_claim`?

Yep!